src/pivotal-ui/components/panels/panels.scss

```diff
@@ -130,175 +157,14 @@ Basic alternate panels are panels with a border and shadow. You can have a panel
 .panel-basic-alt {
   border: $panel-basic-alt-border;
   border-radius: $panel-basic-alt-border-radius;
+  overflow: hidden;
```

Comment:
Here is what I'd do to mitigate risk: put that one piece on it's own special class.
```
.panel-crop {}
```
or maybe
```
.table-wrapper{}
```
Either name seems legit. In that way, we can apply it when we're cropping tables, and not otherwise.